components/contact/Contact.tsx

```diff
@@ -0,0 +1,20 @@
+import ContactHeader from './ContactHeader';
+import ContactForm from './ContactForm';
+import ContactDetails from './ContactDetails';
+const Contact = () => {
+  return (
+    <section className="w-full bg-white-900 px-[1.5rem] py-[3rem] dark:bg-black-200 sm:px-[5.3rem]  2xl:px-[15rem] xlPlus:px-[25rem]">
+      <div className="mx-auto  mt-6  md:max-w-7xl">
+        <ContactHeader />
+      </div>
```

Comment:
You could incorporate the div's styling within ContactHeader, ContactForm, and ContactDetails. This would clean up the code and improve readability. 

 ```
return (
    <section className="w-full bg-white-900 px-[1.5rem] py-[3rem] dark:bg-black-200 sm:px-[5.3rem]  2xl:px-[15rem] xlPlus:px-[25rem]">
     <ContactHeader />
     <ContactForm />
     <ContactDetails />
```